libc/test/src/string/memory_utils/op_tests.cpp

```diff
@@ -246,6 +246,7 @@ int CmpBlockAdaptor(cpp::span<char> p1, cpp::span<char> p2, size_t size) {
   return (int)FnImpl(as_byte(p1), as_byte(p2));
 }
 
+// TODO: should testing of bcmp be moved to libc/test/src/strings/ dir?
```

Comment:
@gchatelet thoughts on this question?

Reply:
It makes sense for unit tests and fuzz tests to move to a `strings` folder but `op_tests` is about testing the low level constructs and I think it's fine to keep them here.

Reply:
24126c4d6da2